ibis/bigquery/tests/conftest.py

```diff
@@ -0,0 +1,60 @@
+import six
+import pytest
+
+import pandas as pd
+import ibis.util
+import ibis.bigquery
+from ibis.bigquery.client import infer_schema_from_df
+
+
+pytest.importorskip('google.cloud.bigquery')
```

Comment:
You don't need this here if you don't `import ibis.bigquery` because pytest will not call fixture functions until after the test module that uses them has been defined, at which point it will hit the `pytest.importorskip('google.cloud.bigquery')` that lives in the test file.